package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@ant-design/icons": "^5.2.6",
+    "@babel/plugin-proposal-private-property-in-object": "^1.0.0",
```

Comment:
What's this? I got an error for this when `npm install`
``` bash
❯ npm install
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @babel/plugin-proposal-private-property-in-object@^1.0.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

Reply:
removed